catalog/to-consul/resource_test.go

```diff
@@ -2291,11 +1281,117 @@ func testService(name string) *apiv1.Service {
 		Status: apiv1.ServiceStatus{
 			LoadBalancer: apiv1.LoadBalancerStatus{
 				Ingress: []apiv1.LoadBalancerIngress{
-					apiv1.LoadBalancerIngress{
+					{
 						IP: "1.2.3.4",
 					},
 				},
 			},
 		},
 	}
 }
+
+func testNodePortService(name string) *apiv1.Service {
```

Comment:
docs for this please. When reading the tests it's nice to quick-definition read the docs to understand what my inputs are.

Reply:
✅ done